src/pages/Home.vue

```diff
@@ -1,6 +1,122 @@
 <template>
   <main>
-    <h1>LevelDB Inspector</h1>
-    <img src="@images/logo.png" alt="App Logo" />
+    <header>
+      <div class="logo"></div>
+    </header>
+    <form>
+      <p>Select path to LevelDB</p>
+      <label class="file db-path">
+        <input type="file" id="file" aria-label="File browser example" />
+        <span class="file-custom"></span>
+      </label>
+      <input type="submit" value="Inspect" />
+    </form>
+    <footer>Copyright 2020 • UncaughtException</footer>
```

Comment:
@jjlumagbas Let's get the year dynamically from the current year, so we don't have to change this value next year.